test/analytics/analytics-on-redshift/lambda/scan-metadata-workflow/store-metadata-into-ddb.test.ts

```diff
@@ -44,151 +44,474 @@ describe('Lambda - store the metadata into DDB from Redshift', () => {
     process.env.REDSHIFT_SERVERLESS_WORKGROUP_NAME = workGroupName;
   });
 
-  test('Store metadata with response FINISHED', async () => {
+  test('Combine and Store metadata with response FINISHED', async () => {
+    const eventDDBCommonData = {
+      id: 'projectId#appId#eventName',
+      month: '#202301',
+      prefix: 'EVENT#projectId#appId',
+      projectId: 'projectId',
+      appId: 'appId',
+      name: 'eventName',
+    };
+
+    const eventParameterDDBCommonData = {
+      id: 'projectId#appId#eventName#propertyName#valueType',
+      month: '#202301',
+      prefix: 'EVENT_PARAMETER#projectId#appId',
+      projectId: 'projectId',
+      appId: 'appId',
+      name: 'propertyName',
+      eventName: 'eventName',
+      category: 'category',
+      valueType: 'String',
+    };
+
+    const userAttributeDDBCommonData = {
+      id: 'projectId#appId#userattributeName#valueType',
+      month: '#202301',
+      prefix: 'USER_ATTRIBUTE#projectId#appId',
+      projectId: 'projectId',
+      appId: 'appId',
+      name: 'userattributename',
+      category: 'category',
+      valueType: 'String',
+    };
+
+    const eventParameterDayCommonData = {
+      category: 'category',
+      eventName: 'eventName',
+      hasData: true,
+      name: 'propertyName',
+      valueType: 'String',
+    };
+
+    const userAttributeDayCommonData = {
+      category: 'category',
+      hasData: true,
+      name: 'userattributename',
+      valueType: 'String',
+    };
+
     const exeuteId = 'Id-1';
     redshiftDataMock.on(DescribeStatementCommand).resolves({
       Status: StatusString.FINISHED,
     });
     redshiftDataMock.on(ExecuteStatementCommand).resolves({ Id: exeuteId });
     dynamoDBMock.on(BatchWriteCommand).resolves({});
+    dynamoDBMock.on(BatchGetCommand)
+      .resolvesOnce({
+        Responses: {
+          ClickstreamAnalyticsMetadata: [
+            {
+              ...eventDDBCommonData,
+              day7: {
+                count: 40,
+                hasData: true,
+                name: 'eventName',
```

Comment:
why not moving event name to root of object?

Reply:
According to @tyyzqmf  data structure design, there is event name in dayN section.
I still added event name in the root section.
@tyyzqmf  if you ead event from root object. I will remove it from dayN section.

Reply:
moved event name to root of object.